contrib/runners/orquesta_runner/tests/unit/test_notify.py

```diff
@@ -348,3 +348,29 @@ def test_cascade_notify_to_tasks(self):
         self.assertEqual(ac_ex_db.status, action_constants.LIVEACTION_STATUS_SUCCEEDED)
         self.assertTrue(notifier.Notifier._post_notify_triggers.called)
         notifier.Notifier._post_notify_triggers.reset_mock()
+
+
+    def test_no_notify_task_list_for_task_with_notify(self):
```

Comment:
If we don't have test cases for various other scenarios already (multiple tasks in a workflow, notify set only for some of them, all of them, etc.), it would be good to add them.

If we already have them, please ignore my comment.

Reply:
Addressed.